Dockerfile.core

```diff
@@ -22,7 +22,10 @@ echo\n\' > /browser && \
     chmod +x /browser
 
 ENV BROWSER /browser
+RUN echo "en_US.UTF-8 UTF-8" >> /etc/locale.gen && \
```

Comment:
Are you sure we need this? Supposedly defining `LANG` should be enough. Please test

Reply:
yes, unfortunately, we do. 
if I remove this (`"en_US.UTF-8 UTF-8" >> /etc/locale.gen`) step and run `dpkg-reconfigure -f noninteractive locales` I have
```
perl: warning: Setting locale failed.
perl: warning: Please check that your locale settings:
	LANGUAGE = (unset),
	LC_ALL = "en_US.UTF-8",
	LANG = "en_US.UTF-8"
    are supported and installed on your system.
perl: warning: Falling back to the standard locale ("C").
locale: Cannot set LC_CTYPE to default locale: No such file or directory
locale: Cannot set LC_MESSAGES to default locale: No such file or directory
locale: Cannot set LC_ALL to default locale: No such file or directory
Generating locales (this might take a while)...
Generation complete.
```

and the problem remains. 
The same If I set `LANGUAGE = "en_US.UTF-8"`.

I take the solution from different comments in `https://github.com/tianon/docker-brew-debian/issues/45#issue-157139109`, but I need to find a right way to compose it.

Reply:
We already run `locale-gen` a few lines above and it does exactly that. I bet that this is a Debian glitch but we need to find the root cause precisely.

Reply:
I can delete run of `locale-gen` because it seems to fail and I was not able to solve our problem using this command.